### ISSUE TITLE
Krypto GDAX exchange

### ISSUE THREAD

**Issue by immanuelgerry** [ctubio/Krypto-trading-bot#854]
HI! can someone help me as i cant connect to GDAX exchange. it says that the API key is invalid.

Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:08.854478 GW COINBASE Warning {"message":"Invalid API Key"}.
12/04 12:54:09.854815 QE Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:10.855105 QE Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:11.855290 QE Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:12.855425 QE Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:13.556714 GW COINBASE FIX Logout, reconnecting in 7s..
12/04 12:54:14.556959 QE Warrrrning: Unable to calculate quote, missing market data.
12/04 12:54:14.800904 GW COINBASE WS Disconnected, reconnecting in 7s..
![whatsapp image 2018-12-04 at 7 54 14 pm](https://user-images.githubusercontent.com/44962291/49489152-4c079a80-f87c-11e8-9195-9576bbdf1127.jpeg)

**Comment by immanuelgerry** [comment]
yes, i have input them correctly. i've tried many times also. please help

**Comment by ctubio** [comment]
then for security reasons, do not search for help here, go to coinbase support team at https://support.coinbase.com/customer/portal/emails/new and talk to them about your error:
```
{"message":"Invalid API Key"}.
```

tell them you are using a bot from https://github.com/ctubio/Krypto-trading-bot that requires ` Trade` and `View` permissions if you consider

this will fix your issue faster and securely

i can tell you the bot works correctly, the problem may be with your account, so ask to coinbase support team
